### PR TITLE
Introducing docker-compose

### PR DESCRIPTION
**Pull request by maxim-tschumak**
This is PR for issue #1. It includes:
* Docker-compose spring profile for storage application
* Docker-compose for swagger-ui, storage and postgres database
* Readme for how to run the application stack locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,11 +47,25 @@ After the crawler loaded the definition it tries to store it in the [storage](st
...
 
-
 ### Installation
 
 Please refer to individual installation instructions of sub-projects for more detailed information and configuration parameters.
 
+
+### Run locally
+
+You can start the whole stack (Swagger UI and Strorage incl. PostgreSQL DB) locally be building the respective sub-projects and running:
```
Comment (fmueller) [inline README.md]: Minor typo: be building -> by building

Comment (maxim-tschumak) [inline README.md]: fixed, thanks!

**Comment by fmueller** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by fmueller** [comment]
👍

**Pull request merged**